.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -131,7 +131,12 @@ module.exports = (actionInfo) => {
       // to the correct versions
       for (const pkgName of pkgDatas.keys()) {
         const { pkg, pkgPath } = pkgDatas.get(pkgName)
-        await exec(`cd ${pkgPath} && yarn pack -f ${pkg}-packed.tgz`, true)
+        // Yarn installed through corepack will not run in pnpm project without this env var set
+        // This var works for corepack >=0.15.0
+        await exec(
+          `cd ${pkgPath} && COREPACK_ENABLE_STRICT=0 yarn pack -f ${pkg}-packed.tgz`,
```

Comment:
You need to use the options parameter of `exec` to make this cross platform compatible

```js
{
  env: {
    'COREPACK_ENABLE_STRICT': '0'
  }
}
```

Reply:
:facepalm: Thanks.

Reply:
Tested with node `19.x` and corepack `0.15.1`.